source/testing/testing-controllers.md

```diff
@@ -119,10 +119,10 @@ test('should modify the post model', function(assert) {
   const postCtrl = ctrl.get('post');
 
   // wrap the test in the run loop because we are dealing with async functions
-  Ember.run(function() {
+  run(function() {
 
     // set a generic model on the post controller
-    postCtrl.set('model', Ember.Object.create({ title: 'foo' }));
+    postCtrl.set('model', EmberObject.create({ title: 'foo' }));
```

Comment:
Thanks @aureliosaraiva, could you add the imports to the appropriate tests as well?  You've got it in your initial change, but subsequent files will need it as well ...